frontend/schema/interface-wpseo-graph-piece.php

```diff
@@ -10,6 +10,12 @@
 	 * An interface for registering Schema Graph Pieces
 	 */
 	interface WPSEO_Graph_Piece {
+		/**
+		 * WPSEO_Graph_Piece constructor.
+		 *
+		 * @param WPSEO_Schema_Context $context A value object with context variables.
+		 */
+		public function __construct( WPSEO_Schema_Context $context );
```

Comment:
I would remove this line. Enforcing the constructor to have a certain interface is restrictive and usually doesn't add a lot of benefit. If a graph piece in the future needs an extra dependency (ver likely) or doesn't use the `$context` at all (rare, but possible) that is not possible.